tests/Unit/Events/LineItemUpdatedTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ *  This program is free software; you can redistribute it and/or
+ *  modify it under the terms of the GNU General Public License
+ *  as published by the Free Software Foundation; under version 2
+ *  of the License (non-upgradable).
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program; if not, write to the Free Software
+ *  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ *  Copyright (c) 2022 (original work) Open Assessment Technologies S.A.
+ */
+
+namespace OAT\SimpleRoster\Tests\Unit\Events;
+
+use OAT\SimpleRoster\Events\LineItemUpdated;
+use PHPUnit\Framework\TestCase;
+
+class LineItemUpdatedTest extends TestCase
+{
+    public function testGetLineItemSlugs(): void
+    {
+        $obj = new LineItemUpdated($list = ['str1', 'str2']);
```

Comment:
let's avoid such constructions. 
doesn't have benefits, but more difficult to read and debug

Reply:
fixed